src/Sylius/Component/Core/Model/AdminUser.php

```diff
@@ -86,4 +88,17 @@ public function setAvatar(?ImageInterface $avatar): void
     {
         $this->setImage($avatar);
     }
+
+    public function isEqualTo(UserInterface $user): bool
+    {
+        if (!$user instanceof AdminUserInterface) {
+            return false;
+        }
+
+        if ($this->isEnabled() !== $user->isEnabled()) {
+            return false;
+        }
+
+        return true;
```

Comment:
```suggestion
        return $this->isEnabled() === $user->isEnabled();
```

Reply:
I'll wait for more opinions. I believe in the context of this method it's better to left it as-is 💃🏻.